docs/api/web-view-tag.md

```diff
@@ -279,10 +279,12 @@ Returns a `String` representing the user agent for guest page.
 
 Injects CSS into the guest page.
 
-### `<webview>.executeJavaScript(code, userGesture)`
+### `<webview>.executeJavaScript(code, userGesture, callback)`
 
 * `code` String
 * `userGesture` Boolean - Default `false`.
+* `callback` Function (optional) - Called after script has been executed.
+  * `result` Array
```

Comment:
Using `Array` type for `result` is confusing, no one knows what is actually returned.